BottleRocket/FileStuff.cs

```diff
@@ -39,37 +39,37 @@ public static bool ROMisLoaded()
         }
         #endregion
 
-        #region Methods for Serializing and Deserializing JSON
-        public static void ExportJson(string json, string path) //multi-purpose!
+        #region Methods for Serializing and Deserializing Toml
+        public static void ExportTOML(string toml, string path) //multi-purpose!
```

Comment:
Since this method has had to get renamed from one specific type of serializer to a different one maybe it should have a more generic name so that in the future you don't have to rename the method again.

Reply:
Renamed it to ExportData. I apologize if the code quality is poor, I am still a C# beginner who has minimal experience with the language.

Reply:
oh don't worry about it, everyone has to start somewhere